presto-product-tests/bin/run_on_docker.sh

```diff
@@ -99,7 +99,7 @@ function stop_docker_compose_containers() {
     stop_application_runner_containers ${ENVIRONMENT}
 
     # stop containers started with "up"
-    environment_compose down
+    environment_compose down -v
```

Comment:
Change the comment to
# stop containers started with "up", removing their volumes